helpdesk_mgmt/views/helpdesk_dashboard_views.xml

```diff
@@ -100,39 +113,6 @@
                                         </div>
                                     </div>
                                 </div>
-                                <div
```

Comment:
Why are you removing this?

Reply:
hand in hand with the other comment (also related to the dropdown dropdown)